index.html

```diff
@@ -240,5 +240,19 @@ <h2 class="about-title">About <br> Myself</h2>
             
         </section>
        </main>
+
+       <footer class="mobile-footer">
+         <h2 class="form-title font-family-poppins">Contact me</h2>
+         <p class="font-family-poppins">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+
+         <form class="mobile-form" action="https://formspree.io/f/mzbqzlen" method="POST">
+            <input type="text" name="name" id="name" placeholder="name" maxlength="30" required>
+            <input type="email" name="mail" id="mail" placeholder="name@gmail.com" required>
```

Comment:
- [Optional] Great Job for adding placeholder 👍, I think it would be better if you use mining full like `enter your name` `enter your email` 👌👍